docker/k8s/lmdb.yaml

```diff
@@ -6,23 +6,19 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      app: lmdb
+      app: avalon-lmdb
   template:
     metadata:
-      name: lmdb
+      name: avalon-lmdb
       labels:
-        app: lmdb
+        app: avalon-lmdb
     spec:
       containers:
-      - image: tcf-dev:latest
+      - image: avalon-lmdb-dev:latest
         name: lmdb-server
         imagePullPolicy: Never
-        envFrom:
-        - prefix: TCF_
-          configMapRef:
-            name: tcf-config
         command: ["/bin/bash"]
-        args: ["$(TCF_HOME)/scripts/lmdb.sh"]
+        args: ["-c", "kv_storage --bind http://0.0.0.0:9090"]
```

Comment:
why do you use 0.0.0.0 instead of localhost?

Reply:
This is to bind to any IPv4 address which is not allocated yet. Container pods IP will be dynamically allocated by K8s. But other pods will communicate to it via service abstraction exposed by this pod